packages/lib/server/queries/teams/index.ts

```diff
@@ -490,3 +507,58 @@ export async function updateNewTeamMemberEventTypes(userId: number, teamId: numb
       })
     ));
 }
+
+export async function addNewMembersToEventTypes({ userIds, teamId }: { userIds: number[]; teamId: number }) {
```

Comment:
We were using `updateNewTeamMemberEventTypes` for each user but this was causing DB connection pool issues. Now we can use `createMany` for creating children event types and new hosts.